CMakeLists.txt

```diff
@@ -39,29 +38,29 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 option(USE_ASIO_STANDALONE "Build with standalone ASIO" ON)
-if (USE_ASIO_STANDALONE)
+if(USE_ASIO_STANDALONE)
     message(STATUS "Using standalone ASIO")
     add_definitions(-DASIO_STANDALONE)
-else ()
+else()
     message(STATUS "Using Boost ASIO")
     find_package(Boost REQUIRED)
-endif (USE_ASIO_STANDALONE)
+endif()
 
 include(TestBigEndian)
-TEST_BIG_ENDIAN(ENDIAN)
-if (ENDIAN)
+test_big_endian(ENDIAN)
```

Comment:
Keep CAP case？

Reply:
ROS `ament_lint_cmake` did't allow mix upper and lower case commands, all other function use lower except this.